mkdocs/docs/HPC/troubleshooting.md

```diff
@@ -479,4 +479,27 @@ The following have been reloaded with a version change:
 This might result in the same problems as mentioned above. When swapping
 to a different cluster, you can run `module purge` to unload all modules
 to avoid problems (see [Purging all modules](../running_batch_jobs/#purging-all-modules)).
+
+### Multi-job submissions on a non-default cluster
+
+When using multi-job submissions, it could be possible you get following error when you try to run it on a non-default cluster.
+
+<pre><code>$ <b> wsub</b>
+/apps/gent/RHEL8/skylake-ib/software/worker/1.6.13-iimpi-2021b/bin/wsub: line 27: 2152510 <b>Illegal instruction</b>     (core dumped) ${PERL} ${DIR}/../lib/wsub.pl "$@"
+</code></pre>
+
+When executing the `module swap cluster/…` command, you are not only changing your environment to submit to the other cluster, but also to use the part of the central software stack that is specific to that cluster. The latter means that you are running a worker on top of a Perl installation that is optimized specifically for the CPUs in that cluster workernodes. That installation is not compatible with the CPUs of the login nodes, which explains the "Illegal instruction" error.
+
+The cluster modules are split up into several "submodules" to help deal with this problem.
+By using `module swap env/slurm/<submit cluster>` instead of `module swap cluster/<submit cluster>` (starting from the default environment where the default cluster is loaded), you can update your environment to submit jobs to the `<submit cluster>`, while still using the software installations that are specific to the default cluster (which are compatible with the login nodes since the default cluster workernodes have the same CPUs).
+
+
+!!! Example
+    if you want to submit a worker job on a different cluster, please use
```

Comment:
```suggestion
    If you want to submit a Worker job to the `donphan`, please use:
   ...
```

and adjust accordingly below, don't use `<submit cluster>` to avoid that people effectively use `<` and `>` in the command...